Marsiple/App/Posts/PostsViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PostsViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class PostsViewController: UIViewController {
+    private let posts = [Post(title: "Title 1", body: "Body 1"),
+                         Post(title: "Title 2", body: "Body 2"),
+                         Post(title: "Title 3", body: "Body 3"),
+                         Post(title: "Title 4", body: "Body 4"),
+                         Post(title: "Title 5", body: "Body 5"),
+                         Post(title: "Title 6", body: "Body 6")]
+    private let postsView = PostsView.autolayoutView()
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        title = "Posts"
```

Comment:
Localize - or, can we completely remove this?

Reply:
This is used by `TabBar` to put a name on a `ViewControllers` tab.

Reply:
Alright. Then we need to `localize`.